store/store.go

```diff
@@ -230,6 +240,32 @@ func (m *manager) insertBlock(ctx context.Context, dbTx *gorm.DB, block *types.B
 			return err
 		}
 		events = append(events, es...)
+		totalTxsFee.Add(totalTxsFee, new(big.Int).Mul(txsFee[receipt.TxHash], new(big.Int).SetUint64(receipt.GasUsed)))
+	}
+
+	minerReward, unclesReward := common.AccumulateRewards(block.Header(), block.Uncles())
+	//insert uncle reward
+	for i, u := range block.Uncles() {
+		events = append(events, &model.Transfer{
+			Address:     model.ETHBytes,
+			BlockNumber: u.Number.Int64(),
+			TxHash:      u.Hash().Bytes(),
+			From:        RewardToUncle.Bytes(),
+			To:          u.Coinbase.Bytes(),
+			Value:       unclesReward[i].String(),
+		})
+	}
+
+	// insert miner reward
+	if !bytes.Equal(block.Coinbase().Bytes(), EmptyAddress.Bytes()) {
```

Comment:
Q: under what circumstances is `block.Coinbase().Bytes()` empty?

Reply:
only........in test 🤣🤣 🤣 
A tremendous amount of work, so i don't want to fix in this PR.